### PR TITLE
Start of the proyect and creation of new articles

### PR DESCRIPTION
**Pull request by mpizzagalli**
<h1> Summary
- Start and creation of the new proyect.
- Implemented feature to create new articles.

<h1> Test Plan
1. Go to http://localhost:3000/articles/new
2. Fill the form fields and save article.
3. Check the article is saved in the db.

<h1> Trello Card

https://trello.com/c/LVPfSJ2w/24-crear-blog-base

### CODE REVIEW

**Comment by mpizzagalli** [comment]
@epintos

File: myapp/app/helpers/application_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ApplicationHelper
```
Comment (epintos) [inline myapp/app/helpers/application_helper.rb:1]: Its a good practice to remove unused and commented code

File: myapp/test/controllers/welcome_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```
Comment (epintos) [inline myapp/test/controllers/welcome_controller_test.rb:1]: Remove the test folder. You will be using something else in a future card

**Comment by epintos** [comment]
@mpizzagalli Good work! Just a few comments for future PR
- Its not a gold rule. But PR should have one commit. Whats the idea of this? PR should be little. In this one you have `76 files changed`. That means you made too many things in this PR, and if you see your commits, you have 4 different things that could be made in 4 differents PR

You PR is :+1: . You can merge it now

**Pull request merged**